### DISCUSSION TITLE
s3 bucket set TTL

### DISCUSSION THREAD

**Discussion by guo-sj** [seaweedfs/seaweedfs#2718] [Q&A]
For now, I've known that setting TTL by `fs.configure` in `weed shell`, but when I tried to configure some path in `/buckets/`, like:
```
fs.configure -locationPrefix=/buckets/my-bucket/guosj/ -ttl=3d -apply
```
I got  the following error log:
```
one s3 bucket goes to one collection and not customizable
```
Does this mean that I can't set TTL for a bucket directory?

**Comment by chrislusf** [comment]
Correct.
